config.json

```diff
@@ -1,9 +0,0 @@
-{   
```

Comment:
i don't think we wanna remove this completely 😅 just not commit/push with your bot token. can we add it back with your new fancy directory?

Reply:
sure we can do that! for sejin we have it removed so thats why my instincts told me to remove it here too haha 😅

Reply:
i think the file is still there though? 🙃

Reply:
omg of course i just realized what you meant, dont mind me, im slow 🙃